src/wasm.rs

```diff
@@ -27,16 +25,17 @@ const SECTION_WC_JWT: &str = "wasmcloud_jwt";
 /// Will return an error if hash computation fails or it can't read the JWT from inside
 /// a section's data, etc
 pub fn extract_claims(contents: impl AsRef<[u8]>) -> Result<Option<Token<Actor>>> {
+    let target_hash = compute_hash(&strip_custom_section(contents.as_ref())?)?;
     let parser = wasmparser::Parser::new(0);
     for payload in parser.parse_all(contents.as_ref()) {
         match payload? {
             wasmparser::Payload::CustomSection(reader) => {
                 if reader.name() == SECTION_JWT || reader.name() == SECTION_WC_JWT {
                     let jwt = String::from_utf8(reader.data().to_vec())?;
                     let claims: Claims<Actor> = Claims::decode(&jwt)?;
-                    let hash = compute_hash_without_jwt(contents.as_ref())?;
+                    //let hash = compute_hash_without_jwt(contents.as_ref())?;
```

Comment:
Nit: remove

Reply:
Old hashes are still not compatible but it doesn't matter. There's a test to prove that old hashes still pass validation checks.

Reply:
Sweet, just remember to remove this commented line before merging